components/script_traits/lib.rs

```diff
@@ -516,6 +510,19 @@ pub struct IFrameLoadInfo {
     pub replace: bool,
 }
 
+/// Specifies the information required to load a URL in an iframe.
+#[derive(Deserialize, Serialize)]
+pub struct IFrameLoadInfoWithData {
```

Comment:
Nit: if we rename `IFrameLoadInfo` to `NewIFrameInfo` then this can be renamed to `IFrameLoadInfo`.